yagocd/session.py

```diff
@@ -75,11 +74,11 @@ def server_version(self):
         This method requests the version from server and caches it
         in internal variable, so other resources could use it.
 
-        :return: server version parsed from `about` page.
+        :return: server version returned from version api endpoint.
         """
         if self.__server_version is None:
-            from yagocd.resources.info import InfoManager
-            self.__server_version = InfoManager(self).version
+            from yagocd.resources.version import VersionManager
+            self.__server_version = VersionManager(self).version
```

Comment:
When I test your branch the VersionManager has no `version` attribute nor does it even have a constructor. What is the mechanism by which this attribute should be added to the object?

Reply:
This should have been taken care with 03794d47f11c47dbe9b0c40a04f9fade5772e6dc.

Reply:
@tfbogdan, can you confirm?

Reply:
Indeed, that deals with it.